frontend/public/components/operator-lifecycle-manager/operand.tsx

```diff
@@ -20,38 +20,50 @@ import { apiVersionForReference, kindForReference, K8sResourceKind, OwnerReferen
 import { ClusterServiceVersionModel } from '../../models';
 import { deleteModal } from '../modals';
 import { RootState } from '../../redux';
+import * as plugins from '../../plugins';
 
 const csvName = () => location.pathname.split('/').find((part, i, allParts) => allParts[i - 1] === referenceForModel(ClusterServiceVersionModel) || allParts[i - 1] === ClusterServiceVersionModel.plural);
 
-const actions = [
-  (kind, obj) => ({
-    label: `Edit ${kind.label}`,
-    href: `/k8s/ns/${obj.metadata.namespace}/${ClusterServiceVersionModel.plural}/${csvName()}/${referenceFor(obj)}/${obj.metadata.name}/yaml`,
-    accessReview: {
-      group: kind.apiGroup,
-      resource: kind.plural,
-      name: obj.metadata.name,
-      namespace: obj.metadata.namespace,
-      verb: 'update',
-    },
-  }),
-  (kind, obj) => ({
-    label: `Delete ${kind.label}`,
-    callback: () => deleteModal({
-      kind,
-      resource: obj,
-      namespace: obj.metadata.namespace,
-      redirectTo: `/k8s/ns/${obj.metadata.namespace}/${ClusterServiceVersionModel.plural}/${csvName()}/${referenceFor(obj)}`,
+const getActions = (ob: K8sResourceKind) => {
```

Comment:
nit: prefer `obj` which is used frequently in console

Reply:
ack